src/lib/booleans.js

```diff
@@ -0,0 +1,119 @@
+function negate(a) {
+  return !a;
+}
+
+function both(a, b) {
+  return a && b;
+}
+
+function either(a, b) {
+  return a || b;
+}
+
+function none(a, b) {
+  return !(a || b);
+}
+
+function one(a, b) {
+  return (a && !b) || (!a && b);
+}
+
+function truthiness(a) {
+  if (a) {
+    return true;
+  }
+
+  return false;
+}
+
+function isEqual(a, b) {
+  if (a === b) {
+    return true;
+  }
+
+  return false;
+}
+
+function isGreaterThan(a, b) {
+  if (a > b) {
+    return true;
+  }
+  return false;
+}
+
+function isLessThanOrEqualTo(a, b) {
+  if (a <= b) {
+    return true;
+  }
+  return false;
+}
+
+function isOdd(a) {
+  if (a % 2 === 1) {
+    return true;
+  }
+  return false;
+}
+
+function isEven(a) {
+  if (a % 2 == 0) {
+    return true;
+  }
+  return false;
+}
+
+function isSquare(a) {
+  /* let root = Math.sqrt(a); 
+  if (Number.isInteger(root)){
+    return true;
+  }
+  return false; */
+
+  if (Number.isInteger(Math.sqrt(a))) {
+    return true;
+  }
+  return false;
```

Comment:
Commented out code like this should always be removed, it just adds noise to the file
```
  return Number.isInteger(Math.sqrt(a));
```